src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Formatters/StandardClassLibraryFormatter.cs

```diff
@@ -35,6 +40,581 @@ public byte[] Deserialize(ref MessagePackReader reader, MessagePackSerializerOpt
         }
     }
 
+    public sealed class SByteArrayFormatter : IMessagePackFormatter<sbyte[]>
```

Comment:
I'm curious why use `sbyte` here? Who uses `sbyte[]`? If there's a perf improvement to be made in writing out an `sbyte[]`  array, wouldn't that also apply to `byte[]`, which is far more popular?

Reply:
[The specialized Formatter](https://github.com/neuecc/MessagePack-CSharp/blob/master/src/MessagePack.UnityClient/Assets/Scripts/MessagePack/Formatters/StandardClassLibraryFormatter.cs#L19) has already existed for `byte[]`. I think that formatter is fastest. I don't need to improve that.

Implemented : `sbyte[]`, `short[]`
Work In Progress : `int[]`, `ushort[]`, `uint[]`

The reason of `sbyte[]` is that its implementation difficulty seemed easy to me compared to others.
I get used to implementing improved formatters.

I will write `int[]` formatter which seems more difficult to implement than `sbyte[]`.

Reply:
well, before you go to any more work, I'd like to feel settled on what you've done so far so you don't waste effort if we're not going to take the PR ultimately anyway. I'm not saying we won't... I'm just saying that since you submitted some, let us review this and understand it enough to justify your continued effort here.